debug/debug_rpc.py

```diff
@@ -75,18 +75,6 @@ def test_debug_startPProf_success(self):
         method = f"{self.ns}_stopPProf"
         Utils.call_ws(self.endpoint, method, [], self.log_path)
 
```

Comment:
Above lines, L21-L76, should be deleted, too.

Reply:
resolve by [c319228](https://github.com/klaytn/klaytn-rpc-tester/pull/13/commits/c319228db76514cca92ba06e0837c6ce25c152b2)